boilerplate/app/components/checkbox/checkbox.tsx

```diff
@@ -52,3 +26,29 @@ export function Checkbox(props: CheckboxProps) {
     </TouchableOpacity>
   )
 }
+
+const $root: ViewStyle = {
+  flexDirection: "row",
+  paddingVertical: spacing.tiny,
+  alignSelf: "flex-start",
+}
+
+const $dimensions = { width: 16, height: 16 }
+
+const $outline: ViewStyle = {
+  ...$dimensions,
+  marginTop: 2, // finicky and will depend on font/line-height/baseline/weather
+  justifyContent: "center",
+  alignItems: "center",
+  borderWidth: 1,
+  borderColor: colors.border,
+  borderRadius: 1,
+}
+
+const $fill: ViewStyle = {
+  width: $dimensions.width - 4,
```

Comment:
I'm guessing we want these 4's to be `spacing.tiny`

Reply:
Fixed width / height wouldn't normally use the spacing scale, but when you're doing it as an offset from screen width or other fixed dimensions, that can be considered spacing since it's usually done to create padding around something